### DISCUSSION TITLE
Number of offers for an ASIN

### DISCUSSION THREAD

**Discussion by arpetris** [amzn/selling-partner-api-models#3473] [General]
I was using the old inventory health report to find the number of sellers in our ASINs. This was more convenient than a call to getCompetitiveOffers as it was one request for all our inventory. The new inventory health report doesn't include this data. Is there another report that includes number of sellers or a better way than using getCompetitiveOffers?

We create affiliate links to Amazon for all our products and want to know when there a no more active Amazon offers so we can remove it from our website.

**Comment by 0xs3n4** [comment]
I would also like to know the total number of offers on the ASIN... I've looked a lot and haven't found it, and I would like more information such as how many FBA/FBM offers we have, and the price of each offer... I'm going through the documentation and I don't find anything..

**Comment by 0xs3n4** [comment]
Hello my friend, look at me here again rsrsrsrs, after many searching in doc and git hub discussions, i find this API
[API-PRINCING-V0](https://developer-docs.amazon.com/sp-api/docs/product-pricing-api-v0-reference)

I assume I didn't consider this option, because we have a more updated PRODUCt-PRINCING, so I didn't look at the old one because I thought it would be discontinued... The lesson is to look at all the versions hehehe
